src/rules/consistent-type-specifier-style.js

```diff
@@ -40,7 +40,8 @@ module.exports = {
   meta: {
     type: 'suggestion',
     docs: {
-      description: 'Enforce or ban the use of inline type-only markers for named imports',
+      category: 'Style guide',
```

Comment:
what's the consequence of adding these categories?

Reply:
This property is not use by ESLint but is commonly used by ESLint plugins for tooling/documentation purposes. In this case, I have used it so the tool can generate the separated lists thanks to `--split-by meta.docs.category`.

Reply:
ok, so you're just matching what was previously in the readme, in the rule metadata?

Reply:
Yep. Rule metadata should be the source of truth.